CONTRIBUTING.md

```diff
@@ -59,33 +43,83 @@ pnpm i
 pnpm run setup
 ```
 
-Note for Windows:
-Use the latest version of [Git Bash](https://gitforwindows.org/)
+> 💡 For Windows users: use the latest version of [Git Bash](https://gitforwindows.org/).
 
 ## Building packages when you make changes
 
 In the root directory:
 
 - `pnpm run setup` will install and build all the packages
 - `pnpm -r run build` (-r for recursive) will build all the packages
-- `pnpm -r run dev` (-r for recursive) will build all the packages without running `tsc` (Fastest)
+- `pnpm -r run dev` (-r for recursive) will build all the packages, without running `tsc`
+- `pnpm run watch` will continuously build any packages that have been modified, without running `tsc` (Fastest)
 
 In a package directory, like `packages/client`:
 
 - `pnpm run build` will build the package
 - `pnpm run dev` will build the package without running `tsc` (Fastest)
 
-Note: Our builder is ESbuild
+> 💡 Our builder is built on top of esbuild
 
 ## Prisma Client
 
 ### First contribution
 
-1. `cd packages/client`
-1. `ts-node fixtures/generate.ts ./fixtures/blog/ --skip-transpile`
-1. `cd fixtures/blog`
-1. `npx prisma db push --skip-generate` will create the database structure
-1. `ts-node main`
+Create a reproduction folder for developing, trying a new feature, or a fix.
+
+#### Setting up a locally-linked development folder
+
+Set up a local project that will be linked to the local packages.
+
+```sh
+cd reproductions && pnpm install
+# Copy a template from the reproduction folder
+cp -r basic-sqlite my-repro && cd my-repro
+# Ensure that the db and the schema are synced
+pnpx prisma db push --skip-generate
+# Do some code changes, compile, then try it out
+pnpx prisma generate && pnpx ts-node index.ts
```

Comment:
```suggestion
# Re-generate the Prisma Client
pnpx prisma generate
# Run the script
pnpx ts-node index.ts
```
Maybe cleaner/simpler to cut into their own step

Reply:
I prefer to keep it in one command to be able to re-run the whole thing. Maybe I must make it more explicit that re-generate is needed every time?

Reply:
Added a new suggestion to improve the comment